packages/frontend/index.html

```diff
@@ -9,6 +9,13 @@
     />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Arbeidsflate</title>
+      <!-- Injected Config Script -->
+    <script>
```

Comment:
I think it's an ok temporary solution for application insight key, but other configuration should be accessible through an endpoint (BFF), so I would drop __CONFIG__ and just leave at at one level.